trestle/cli.py

```diff
@@ -41,20 +42,21 @@ class Trestle(CommandPlusDocs):
     """Manage OSCAL files in a human friendly manner."""
 
     subcommands = [
-        InitCmd,
-        CreateCmd,
-        SplitCmd,
-        AuthorCmd,
-        MergeCmd,
-        ReplicateCmd,
         AddCmd,
-        RemoveCmd,
-        ValidateCmd,
+        AssembleCmd,
```

Comment:
As you may have realized the previous order was somewhat based on a natural usage flow that was similar to cli.md.  But alphabetical is fine.